### ISSUE TITLE
[Feature] Change Portal Color

### ISSUE THREAD

**Issue by NoName50** [p2sr/SourceAutoRecord#233]
Addresses are included in the cheat table below
Also included [LUA scripts](https://github.com/p2sr/SourceAutoRecord/files/15213540/portal2_portalcolor_release_1.1EN.zip) to change colors to achieve the effect in the [Krzyhau video](https://youtu.be/4xB08fw5BYc).

Convar suggestions:
```cpp
Variable hud_sp_quickinfo_leftcolor("hud_sp_quickinfo_leftcolor", "64 160 255", "Changes the color of the singleplayer left quickinfo hud.\n");
Variable hud_sp_quickinfo_rightcolor("hud_sp_quickinfo_rightcolor", "255 160 32", "Changes the color of the singleplayer right quickinfo hud.\n");
Variable cl_sp_portalonecolor("cl_sp_portalonecolor", "0 60 255", "Changes the color of the singleplayer primary portal.\n");
Variable cl_sp_portaltwocolor("cl_sp_portaltwocolor", "233 78 2", "Changes the color of the singleplayer second portal.\n");
Variable cl_coop_blue_portalonecolor("cl_coop_blue_portalonecolor", "32 128 210", "Changes the color of Blue's primary portal.\n");
Variable cl_coop_blue_portaltwocolor("cl_coop_blue_portaltwocolor", "19 0 210", "Changes the color of Blue's second portal.\n");
Variable cl_coop_orange_portalonecolor("cl_coop_orange_portalonecolor", "255 180 32", "Changes the color of Orange's primary portal.\n");
Variable cl_coop_orange_portaltwocolor("cl_coop_orange_portaltwocolor", "57 3 3", "Changes the color of Orange's second portal.\n");
```
(I was trying to learn C++, but it's too hard for me😞)

**Comment by ThisAMJ** [comment]
see also https://discord.com/channels/146404426746167296/811780246608281650/1011584350229373019

**Comment by ThisAMJ** [comment]
Mostly implemented by 5541017, but changing the color of sp portals is yet to be done.
